packages/server/src/types/accounts-server-options.ts

```diff
@@ -24,4 +25,6 @@ export interface AccountsServerOptions {
   siteUrl?: string;
   prepareMail?: PrepareMailFunction;
   sendMail?: SendMailType;
+  tokenCreator?: TokenCreator;
+  createSessionTokenOnRefresh?: boolean;
```

Comment:
Can you add a comment to explain this option please?

Reply:
Would the name `createNewSessionTokenOnRefresh` be better? (not so sure what do you think?)